test/graphdb.test.ts

```diff
@@ -0,0 +1,97 @@
+import {GraphDBClient} from '../src/graphdb';
+import {Dataset} from '../src/dataset';
+import {QueryEngine} from '@comunica/query-sparql';
+// eslint-disable-next-line node/no-unpublished-import
+import {GenericContainer, StartedTestContainer} from 'testcontainers';
+import factory from 'rdf-ext';
+import axios from 'axios';
+// eslint-disable-next-line node/no-unpublished-import
+import FormData from 'form-data';
+import * as fs from 'fs';
+import * as path from 'path';
+import {fileURLToPath} from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const graphDb = new GenericContainer(
+  'ontotext/graphdb:10.5.1'
+).withExposedPorts(7200);
+let startedGraphDb: StartedTestContainer;
+
+const host = (container: StartedTestContainer) =>
+  `http://localhost:${container.getMappedPort(7200)}`;
+
+describe('GraphDBClient', () => {
+  let client: GraphDBClient;
+
+  beforeAll(async () => {
+    startedGraphDb = await graphDb.start();
+
+    const formData = new FormData();
+    formData.append(
+      'config',
+      fs.createReadStream(
+        path.join(__dirname, './assets/graphdb-repo-config.ttl')
```

Comment:
Should this be checked in?

Reply:
yep, that explains the not working test :)